packages/named/index.js

```diff
@@ -1,19 +1,18 @@
-var compose = require('@stamp/compose');
+'use strict';
```

Comment:
This module is deprecated. :) We can pretty much remove it from the repo.

Reply:
Should the npm package be [deprecated](https://docs.npmjs.com/deprecating-and-undeprecating-packages-or-package-versions)?

Can you update the following files with deprecation reasoning?

- `packages/named/index.js`
- `packages/named/README.md`

Reply:
@koresar ?

Reply:
Sorry. Don't know how to do that. Have no time currently. Sorry again. Life.

Reply:
No worries. We can do this later.